iterm2_dwim/__init__.py

```diff
@@ -5,12 +5,13 @@
 import sys
 import traceback
 
-from iterm2_dwim.editors import emacs
+from iterm2_dwim.editors import emacs, sublime
+from iterm2_dwim.settings import editor
```

Comment:
nit: I prefer importing the `settings` module, then it is clear in the code that something like `settings.emacsclient_path` is a setting.